src/lib/ChatSuggestions/properties.ts

```
@@ -10,8 +39,28 @@ export type MandatoryChatSuggestionsProperties = {
   disabled?: boolean;
+  layout?: ChatSuggestionsLayout;
+  direction?: ChatSuggestionsDirection;
+  /** Render at most this many chips. Omit to render every item. */
+  maxVisible?: number;
+  /**
+   * Suppresses the whole row without unmounting it — for the window between asking
+   * and answering, where stale suggestions would invite a second wrong question.
+   */
+  loading?: boolean;
+  /** Custom mark per chip; receives the item's resolved icon string, if any. */
+  icon?: Snippet<[string | undefined, number]>;
```

ESLint

[error] undefined type is not allowed. Use null instead.

(no-restricted-syntax)